lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Tady to row vypadá jako tuple, ale přitom není. To by asi stálo za to osvětlit, ale nemyslím implementaci.</p><p>Tedy že je to objekt, který je udělaný tak, aby se s ním dalo pracovat jako s listem i dictem.</p><p> </p><p>Mohl by se tu vysvětlit ten LIMIT nebo to prostě vyřešit v pythonu.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>